src/dvc_render/vega_templates.py

```diff
@@ -612,6 +614,35 @@ class LinearTemplate(Template):
             },
             "color": {"field": "rev", "type": "nominal"},
         },
+        "params": [
+            {
+              "name": "points",
+              "value": 0,
+              "bind": {
+                "input": "radio",
+                "options": [30, 0],
+                "labels": ["show", "hide"]
+              }
+            },
+            {
+                "name": "smooth",
```

Comment:
does it mean we can deprecate smooth template after this? 

I wish we could have done a bit better UI for these sliders in VS Code also - they will be taking half of the plot size I think ...

Reply:
Yes. I also pushed an update to make the smooth template the same as linear. Added an item to https://github.com/iterative/dvc/issues/7093.

Reply:
> I wish we could have done a bit better UI for these sliders in VS Code also - they will be taking half of the plot size I think ...

I think it might should be possible to have custom sliders and bind them to the vega plot (https://vega.github.io/vega/docs/signals/#bind-external) , it doesn't sound trivial though 😅